examples/tide/main.rs

```diff
@@ -56,8 +56,13 @@ async fn index_view(mut request: tide::Request<AppState>) -> tide::Result {
     Ok(askama_tide::into_response(&IndexTemplate {}, "html"))
 }
 
+fn username_from_request(request: &tide::Request<AppState>) -> String {
```

Comment:
Is there a reason this has to be a function? It's only used in one place ...

Reply:
Sorry—I just forgot to clean this up. This was me playing around to see if I could get the error in the example, but I wasn't able to because the tide router is smart enough to not route `/register/` to the route set to `/register/:username`.